src/NuGet/Microsoft.Net.Compilers.netcore.nuspec

```diff
@@ -60,6 +60,8 @@
         </dependencies>
     </metadata>
     <files>
+        <file src="$emptyDirPath$/_._" target="ref/dotnet" />
```

Comment:
When testing these updates I made a test project. Adding the Net.Compilers.netcore package added the CodeAnalysis assemblies and all their references to my project. The purpose of the package was only provide compilers for runtime use, not CodeAnalysis for APIs.